src/scripts/events/domEvents.js

```diff
@@ -0,0 +1,18 @@
+import { deleteCustomer, getCustomers } from '../../api/customerData';
+import { showCustomers } from '../components/pages/customers';
+
+// Customer card events
+const domEvents = () => {
+  document.querySelector('#main-container').addEventListener('click', (e) => {
+    if (e.target.id.includes('delete-customer-btn')) {
+      const [, firebaseKey] = e.target.id.split('--');
+      deleteCustomer(firebaseKey).then(() => {
+        getCustomers().then((customerArray) => {
+          showCustomers(customerArray);
```

Comment:
so, currently in the firebase, customer sample data is not in a division called customers. so when I run this code, there are never any customers. I'm not sure that it matters super-much? because a page to view the customers is not in the mvp, and we'll be generating customers based on user input. all that being said, calls look great to me.